### PR TITLE
570 highres linecolindicator

### PR DESCRIPTION
**Pull request by cubap**

### CODE REVIEW

File: web/js/transcribe.js [hunk 1]
```diff
@@ -6886,12 +7068,12 @@ function setDirectionForElements() {
...
-function checkParsingReroute() {
-    if (getURLVariable('liveTool') == "parsing") {
-        setTimeout(function() {
+function checkParsingReroute(){
+    if(getURLVariable('liveTool') == "parsing"){
+        setTimeout(function () {
             var replaceURL = replaceURLVariable("liveTool", "none");
             window.history.pushState("", "T&#8209;PEN 2.8 Transcription", replaceURL);
-            if (tpen.user.isMember || tpen.project.permissions.allow_public_modify || tpen.project.permissions.allow_public_modify_line_parsing) {
+            if(tpen.user.isAdmin || tpen.project.permissions.allow_public_modify || tpen.project.permissions.allow_public_modify_line_parsing){
```
Comment (thehabes) [inline web/js/transcribe.js:7076]: Why this change

**Review by thehabes — CHANGES_REQUESTED** [review]

File: web/js/transcribe.js [hunk 2]
```diff
@@ -2819,7 +2883,9 @@ function zoomBookmark(zoomOut) {
             "top": (imgDims[3] - bookmark.height()) * zoomRatio,
             "width": imgDims[1] * zoomRatio / WRAPWIDTH * 100 + "%"
         });
+        $("#imgTop").css("background-image", $(".activeLine").css("background-image"))
```
Comment (cubap) [inline web/js/transcribe.js:2886]: finally something I was supposed to add.

File: web/js/transcribe.js [hunk 3]
```diff
@@ -3119,12 +3190,15 @@ function makeOverlayDiv(thisLine, originalX, cnt) {
     if (thisLine.attr("data-answer") !== undefined && thisLine.attr("data-answer") !== "") {
         hasTrans = true;
     }
-    var lineOverlay = "<div class='parsing' lineid='" + (parseInt(cnt) - 1) + "' style='top:" +
-        newY + "%;left:" + newX + "%;height:" +
-        newH + "%;width:" + newW + "%;' lineserverid='" +
-        thisLine.attr('lineserverid') + "'linetop='" +
-        Y + "'lineleft='" + X + "'lineheight='" +
-        H + "'linewidth='" + W + "' hastranscription='" + hasTrans + "'></div>";
+    if ($(".transcriptionImage").attr("src").includes('/full/full')) {
```
Comment (cubap) [inline web/js/transcribe.js:3193]: hey! I added this on purpose!

File: web/js/transcribe.js [hunk 4]
```diff
@@ -4065,172 +4124,180 @@ function addchar(theChar, closingTag) {
...
-    else if (e.createTextRange) {
-        e = e.createTextRange();
-        e.collapse(true);
-        e.moveEnd('character', pos);
-        e.moveStart('character', pos);
-        e.select();
-    }
-    return wrapped;
-}
+//function setCursorPosition(e, position) {
```
[the remark this answers is not in the document]
Comment (cubap) [reply web/js/transcribe.js:4161]: The greatest of these is Love.
